sdk/keyvault/azure-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/KeyProperties.java

```diff
@@ -38,7 +33,7 @@
      * include: 'EC', 'EC-HSM', 'RSA', 'RSA-HSM', 'oct'.
      */
     @JsonProperty(value = "kty")
-    private KeyType keyType;
+    private CertificateKeyType keyType;
```

Comment:
should the variable name be certificateKeyType?

Reply:
nope, the key type is the same as in keys package.
This would mean adding certificate prefix to all the variables.
Its a certificate policy, that part is implied.